### DISCUSSION TITLE
Wireless connection

### DISCUSSION THREAD

**Discussion by himlastar** [BimmerGestalt/AAIdrive#838] [General]
Is it possible to establish a wireless connection without usb cable?

**Comment by himlastar** [comment]
Using something like this?

https://www.amazon.de/T2U-802-11ac-encryption-standard-supports/dp/B07KRCW6LZ/ref=mp_s_a_1_3?crid=1OIS5ZU5RXETD&dib=eyJ2IjoiMSJ9.P2jS0GWL_wMh7XWEtx36fgrXJV6JhDtH4zBiTIHifbxPGvToth0WDFabeKvoMZO61Hyx90xemkf-onmMhNpUSy4jEySi_QsMQeX1f-BC_MoJ_Uzbi_gG9HoIwtsaWMAc9pcGY5B1Zq9se1icjJrXDUDzqgvFuCIiwW4oDrIg9VhaaT-JDNllqCMDN7OQ7DysNW8ErK4UXy3YL-g_1Il20Q.R_MIP6sL_DxsbfEUlgdGy2SSiOUe_0SGhJBdblK9eFw&dib_tag=se&keywords=usb+a+wireless&nsdOptOutParam=true&qid=1732826408&sprefix=usb+a+wireles%2Caps%2C126&sr=8-3

**Comment by hufman** [comment]
I don't know of any way to run the car app protocol over a wireless connection if the car doesn't already support it. The car is expecting the Android Open Accessory Protocol in the USB port, and won't be looking anywhere else for the protocol.

**Reply by himlastar** [reply]
Is there a way to retrofit something into the nbt so it works wireless? I just want pictures when playing music, for example my wife's iPhone works fine, but my android doesn't.

**Reply by hufman** [reply]
As far as I know, there isn't anything besides upgrading the entire unit to IDrive 5 or 6 with new hardware. Even then, the IDrive will still ignore coverart over Bluetooth from Android.
